### PR TITLE
Add polyfill module

### PR DESCRIPTION
**Pull request by LucaMele**

### CODE REVIEW

File: src/components/05-utils/polyfill/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# AXA Patterns Library Polyfill
+
+This package includes polyfills for all AXA B2C Browsers. It includes 2 types of bundles:
+
+- Bundle for IE11 + WebComponents
+- Bundle for modern browser that don't support yet WebComponents.
```
Comment (raphaellueckl) [inline src/components/05-utils/polyfill/README.md]: Correct: "Bundle for modern browser that don't support WebComponents yet."

File: src/components/05-utils/polyfill/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+## Usage
+
+Install the package:
+
+```bash
+npm install @axa-ch/patterns-library-polyfill
+```
+
+For IE11 + WebComponents support in a HTML only page:
```
Comment (raphaellueckl) [inline src/components/05-utils/polyfill/README.md]: "HTML-only"

File: src/components/05-utils/polyfill/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+For IE11 + WebComponents support in a frontend stack:
+
+```js
+import '@axa-ch/patterns-library-polyfill';
+
+// rest of your code here
+```
+
+For Modern Browser support in a HTML only page:
```
Comment (raphaellueckl) [inline src/components/05-utils/polyfill/README.md]: "HTML-only"

File: src/components/05-utils/polyfill/README.md (hunk 1 above)
Comment (LucaMele) [reply src/components/05-utils/polyfill/README.md]: Done

File: src/components/10-atoms/button-link/index.js [hunk 4]
```diff
@@ -75,11 +74,12 @@ class AXAButtonLink extends LitElement {
...
         <div class="a-button-link__flex-wrapper">
-          <slot></slot>
-          ${icon &&
-            html`
-              <axa-icon class="a-button-link__icon" icon="${icon}"></axa-icon>
-            `}
+          <slot></slot> ${
+            icon &&
+              html`
+                <axa-icon class="a-button-link__icon" icon="${icon}"></axa-icon>
```
Comment (raphaellueckl) [inline src/components/10-atoms/button-link/index.js]: Attention! This writes "false" to the DOM if icon is falsy.

File: src/components/10-atoms/button/index.js [hunk 5]
```diff
@@ -76,11 +75,12 @@ class AXAButton extends LitElement {
         @click="${this.onClick}"
       >
         <div class="a-button__flex-wrapper">
-          <slot></slot>
-          ${icon &&
-            html`
-              <axa-icon class="a-button__icon" icon="${icon}"></axa-icon>
-            `}
+          <slot></slot> ${
+            icon &&
```
Comment (raphaellueckl) [inline src/components/10-atoms/button/index.js]: Attention! This writes "false" to the DOM if icon is falsy.

File: src/components/05-utils/polyfill/README.md (hunk 2 above)
Comment (LucaMele) [reply src/components/05-utils/polyfill/README.md]: done

File: src/components/10-atoms/button-link/index.js (hunk 4 above)
Comment (LucaMele) [reply src/components/10-atoms/button-link/index.js]: This is not my code, prettier formated it like that so i dont know the logic

File: src/components/10-atoms/button/index.js (hunk 5 above)
Comment (LucaMele) [reply src/components/10-atoms/button/index.js]: no changes here from my side, was prettier

**Comment by raphaellueckl** [comment]
Okidoki

**Pull request merged**